rpcd/playbooks/roles/rpc_maas/templates/container_storage_checks.yaml.j2

```diff
@@ -0,0 +1,20 @@
+{% set label = "container_storage_check" %}
+{% set check_name = label+'--'+ansible_hostname %}
+type        : agent.plugin
+label       : "{{ check_name }}"
+period      : "{{ maas_check_period_override[label] | default(maas_check_period) }}"
+timeout     : "{{ maas_check_timeout_override[label] | default(maas_check_timeout) }}"
+disabled    : "{{ (check_name | match(maas_excluded_checks_regex)) | ternary('true', 'false') }}"
+details     :
+    file    : run_plugin_in_venv.sh
+    args    : ["{{ maas_plugin_dir }}container_storage_check.py", "--thresh", "{{ percent_used_critical_threshold }}"]
+alarms      :
+container_storage_percent_used_critical:
```

Comment:
Need to push container_storage_percent_used_critical section to nest under alarms, seems like it wasn't picking up until I moved it over.